logstash-core-event-java/src/test/java/com/logstash/KeyNodeTest.java

```diff
@@ -0,0 +1,27 @@
+package com.logstash;
+
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+
+public class KeyNodeTest {
```

Comment:
can `value` that is extracted from the keynode be `null`? if so, I think `toString` on it would fail. should we add a test for this if it is possible?